src/templates/index.jsx

```diff
@@ -151,6 +151,11 @@ const Layout = React.createClass({
             this.noteListTop = noteList.scrollTop;
         }
 
+        // set the list height if it's not set yet
```

Comment:
could we explain why this could be the case? it seems obvious from the code _that_ this is setting it but what I would care about is _why didn't it set it?_

something like…
> if the panel is closed when the component mounts then the calculated height will be zero because it's hidden. we need to make sure that when the panel opens, if we found that zero value, that we appropriately set the real height when actually rendered.